packages/waggle/src/storage/MessagesAccessController.ts

```diff
@@ -0,0 +1,26 @@
+import AccessController from 'orbit-db-access-controllers/src/access-controller-interface'
+import { stringToArrayBuffer } from 'pvutils'
+import { ChannelMessage } from '@zbayapp/nectar'
+import { verifySignature } from '@zbayapp/identity'
+
+const type = 'messagesaccess'
+
+export class MessagesAccessController extends AccessController {
+  static get type () { return type }
+
+  async canAppend(entry) {
+    const message: ChannelMessage = entry.payload.value
+    const signature = stringToArrayBuffer(message.signature)
+    const verify = await verifySignature(signature, message.message, null, message.pubKey)
```

Comment:
```
  const verify = await verifySignature(signature, message.message, null, message.pubKey)
```
here I'd suggest having a mapping of ` { pubKey: CryptoKey }` on this (as a cache). This way we are caching instances of CryptoKey and not "paying" for constructing it for every single message

Reply:
I don't fully get it. We have no power over passing data to this method (I guess). It fires automatically just before saving data to db, e.g. after replication so where can we do any mapping?